### PR TITLE
Setup some test corrections for blue

### PR DESCRIPTION
**Pull request by radtriste**
Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [ ] Pull Request title is properly formatted: `[KOGITO|RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

File: tests/features/bamoe-kogito-builder-jvm.feature [hunk 1]
```diff
@@ -11,7 +11,7 @@ Feature: rhpam-kogito-builder-rhel8 feature.
     And the image should contain label io.k8s.display-name with value IBM build of Kogito builder based on Quarkus or SpringBoot
     And the image should contain label io.openshift.tags with value ibm-bamoe-kogito,builder,kogito,quarkus,springboot
     And the image should contain label io.openshift.s2i.assemble-input-files with value /home/kogito/bin
-    And the image should contain label com.ibm.component with value ibm-bamoe-8-builder-rhel8-container
+    And the image should contain label com.redhat.component with value rhpam-7-kogito-builder-rhel8-container
```
Comment (sutaakar) [inline tests/features/bamoe-kogito-builder-jvm.feature:14]: Is this expected? Or is it some workaround for now?

Comment (radtriste) [reply tests/features/bamoe-kogito-builder-jvm.feature:14]: bug was introduced in https://github.com/kiegroup/kogito-images/commit/5a0465ea3dadffe25e5caa59ecc633e80eacb127#diff-5f131911db7841f58e02a80cc72af72eef57b8585879cdf4a882502b938441b3L14
I am just reverting it ;)

Comment (sutaakar) [reply tests/features/bamoe-kogito-builder-jvm.feature:14]: ok

**Review by sutaakar — APPROVED** [review]

**Review by spolti — APPROVED** [review]

**Comment by radtriste** [comment]
jenkins retest this

**Comment by kie-ci1** [comment]
**PR job** `#17` was: **UNSTABLE**
Possible explanation: This should be test failures

Please look here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images/17/display/redirect

**Test results:**
- PASSED: 41
- FAILED: 1

Those are the test failures: 
<details>
<summary><a href="https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/KIE/job/kogito/job/1.13.x-blue/job/pullrequest/job/kogito-images/17/testReport/features.image.kogito-builder/kogito-builder image tests/Verify_that_the_Kogito_Quarkus_Serverless_Workflow_Extension_is_building_the_service_properly/">features.image.kogito-builder.kogito-builder image tests.Verify that the Kogito Quarkus Serverless Workflow Extension is building the service properly</a></summary>
S2I build failed, check logs!
</details>

**Comment by radtriste** [comment]
issue not related

**Pull request merged**